example_external/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>generate_parameter_library_example_external</name>
-  <version>0.1.0</version>
+  <version>0.4.0</version>
```

Comment:
Note I changed this package to just track the same version as the rest of the repo.

I had overlooked that in reviewing the original PR.